### PR TITLE
OS X compatibility with Grpc 0.12

### PR DESCRIPTION
**Pull request by Rastusik**
Build process compatible with OS X after integrating Grpc 0.12

### CODE REVIEW

**Comment by Rastusik** [comment]
I signed it!

File: doc/GettingStarted.md [hunk 1]
```diff
@@ -245,6 +246,16 @@ In addition, Vitess requires the software and libraries listed below.
     If your machine requires a proxy to access the Internet, you will need
     to set the usual environment variables (e.g. `http_proxy`,
     `https_proxy`, `no_proxy`).
+    
+    Note:
+        On OS X, if you have already installed grpc and protobuf using homebrew, you need to unlink protobuf while the ./bootstrap.sh
+        script is running.
+        
+        ```sh
+        brew unlink grpc/grpc/google-protobuf
```
Comment (michael-berlin) [inline doc/GettingStarted.md]: Does it make sense to put these two commands in bootstrap.sh instead?

That should be more user friendly than adding these steps to the documentation.

File: doc/GettingStarted.md [hunk 2]
```diff
@@ -263,6 +274,13 @@ In addition, Vitess requires the software and libraries listed below.
     . ./dev.env
     make build
     ```
+    
+    Note:
+        On OS X, if you can link the protobuf library back.
```
Comment (michael-berlin) [inline doc/GettingStarted.md]: Remove "if" here from sentence?

As I mentioned above, I think I would prefer if this is part of bootstrap.sh instead.

File: doc/GettingStarted.md (hunk 1 above)
Comment (Rastusik) [inline doc/GettingStarted.md]: good point, done

**Comment by michael-berlin** [comment]
LGTM

I'm merging it now.

**Pull request merged**